src/core/scene.ts

```diff
@@ -932,6 +935,261 @@ export class Scene {
+        return new Promise((resolve, reject) => {
+          Scene.getActiveScene()
+          .then(activeScene => {
+            return activeScene.removePreset(preset);
+          }).then(setFlag => {
+            resolve(setFlag);
+          }).catch(err => reject(err))
+        })
+      }
+
+      Scene._liveScene.getPresetTransitionEasing = (): Promise<string> => {
+        return new Promise((resolve, reject) => {
+          Scene.getActiveScene()
+          .then(activeScene => {
+            return activeScene.getPresetTransitionEasing();
+          }).then(easing => {
+            resolve(easing);
+          }).catch(err => reject(err))
+        })
+      }
+
+      Scene._liveScene.setPresetTransitionEasing = (presetTransitionEasing: string): Promise<boolean> => {
+        return new Promise((resolve, reject) => {
+          Scene.getActiveScene()
+          .then(activeScene => {
+            return activeScene.setPresetTransitionEasing(presetTransitionEasing);
+          }).then(setFlag => {
+            resolve(setFlag);
+          }).catch(err => reject(err))
+        })
+      }
+
+      Scene._liveScene.getPresetTransitionTime = (): Promise<number> => {
+        return new Promise((resolve, reject) => {
+          Scene.getActiveScene()
+          .then(activeScene => {
+            return activeScene.getPresetTransitionTime();
+          }).then(time => {
+            resolve(time);
+          }).catch(err => reject(err))
+        })
+      }
+
+      Scene._liveScene.setPresetTransitionTime = (presetTransitionTime: number): Promise<boolean> => {
+        return new Promise((resolve, reject) => {
+          Scene.getActiveScene()
+          .then(activeScene => {
+            return activeScene.setPresetTransitionTime(presetTransitionTime);
+          }).then(setFlag => {
+            resolve(setFlag);
+          }).catch(err => reject(err))
+        })
+      }
+    }
+    return Scene._liveScene;
+  }
+
+  /**
+   * param: (value?: number | Scene)
+   * ```
+   * return: Promise<any>
+   * ```
+   *
+   * Adds this scene as a source to the current scene by default.
+   * Accepts an optional parameter value, which, when supplied,
+   * points to the scene where item will be added instead.
+   * If ready config {listenToItemAdd: true} it returns item id,
+   * else returns boolean.
+   *
+   * Note: There is yet no way to detect error responses for this action.
+   */
+  addAsSource(value?: number | Scene): Promise<any> {
+    return new Promise((resolve, reject) => {
+      if (!versionCompare(getVersion()).is.lessThan(sceneSourceVersion)) {
+        checkSplitmode(value).then((scenePrefix) => {
+
+        const sceneToAdd = new JXON();
+        sceneToAdd.tag = 'item';
+        sceneToAdd['item'] = this._uid;
+        sceneToAdd['name'] = this._name;
+        sceneToAdd['type'] = '14'; // type LIVE
+        sceneToAdd['selfclosing'] = true;
+        const sceneXML = XML.parseJSON(sceneToAdd);
```

Comment:
Minor thing. Use some sort of auto formatter like prettier. The block of code above are not indented properly.

Reply:
Done:
- indentation
- used the ItemTypes enum
- added doc comments to live scene object